### ISSUE TITLE
Contributors list

### ISSUE THREAD

**Issue by rozPierog** [rozPierog/Cofi#190]
**Is your feature request related to a problem? Please describe.**
Cofi had a lot of contributions from community members over the years, and it would only be fair to have a section in the app with their names and links to their GitHub pages

**Describe the solution you'd like**
Page in settings with contributor name, nick, and a link to their GH page. Autogenerated from a file that's generated by GH action that runs on every commit on a `main` branch

**Describe alternatives you've considered**

**Additional context**

**Comment by flufficat** [comment]
There is already a section on the GitHub page with the contributors pre-filled https://github.com/rozPierog/Cofi/graphs/contributors
Perhaps a link to this would be simpler

**Comment by rozPierog** [comment]
Yup, I know that, but I want to immortalize all of them in the app. If I do it for the dependencies that I use it's only fair to do it for the contributors
